packages/bootleg-tokens/contracts/ISharedRoyaltyTokenEnumerable.sol

```diff
@@ -0,0 +1,36 @@
+pragma solidity ^0.5.3;
+
+/**
+ * @title Shared Royalty NFT Enumerable interface for franchisors
+ */
+contract ISharedRoyaltyTokenEnumerable {
+
+    function totalSupply() public view returns (uint256);
+    function tokenOfOwnerByIndex(address owner, uint256 index) public view returns (uint256 tokenId);
+
+    function tokenByIndex(uint256 index) public view returns (uint256);
+
+    /**
+    * @notice Gets the total number of Franchisees given a Token ID
+    * @param _tokenId The identifier of the NFT
+    * @return uint256 The number of franchisees
+    */
+    function tokenFranchiseesTotal(uint256 _tokenId) public view returns (uint256 count);
+
+    /**
+    * @notice Gets the token franchisee for a given token at a specific index
+    * @param _tokenId The identifier of the NFT
+    * @param _index The index, should be less than total franchisees for the token
+    * @return address The address of the franchisee
+    */
+    function tokenFranchiseesByIndex(uint256 _tokenId, uint256 _index) public view returns (address owner);

```

Comment:
I realized later that `tokenId` in the ERC721 is traditionally last. I'm updating that in another pr for the first interface.

<img src="https://derpicdn.net/img/2012/11/6/143753/full.png" width=200 />